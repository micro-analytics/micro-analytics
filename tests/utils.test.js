const { queryToObject } = require('../src/utils')

describe('utils', () => {
  it("queryToObject should convert string into object", () => {
    expect(queryToObject('k:v,k2:v2')).toMatchSnapshot()
  })
})
