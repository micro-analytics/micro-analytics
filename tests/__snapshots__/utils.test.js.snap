// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils queryToObject should convert string into object 1`] = `
Object {
  "k": "v",
  "k2": "v2",
}
`;
